src/ManagedServices/ManagedServices/ManagedServices.csproj

```diff
@@ -1,14 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <PsModuleName>ManagedServices</PsModuleName>
+    <AssemblyName>s</AssemblyName>
```

Comment:
Why are you adding an empty xml element?

Reply:
That was unintended. I removed it.